src/Computer.js

```diff
@@ -0,0 +1,81 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Lotto from "./Lotto.js";
+import { LOTTO_PICK, MIN_PURCHASE_AMOUNT, WIN_PRIZE } from "./constant.js";
+
+class Computer {
+  printPurchaseCount(purchaseAmount){
+    this.purchaseCount = Math.floor(purchaseAmount / MIN_PURCHASE_AMOUNT);
+  
+    Console.print(`\n${String(this.purchaseCount)}개를 구매했습니다.`);
+  }
+
+  makeLottoList(){
+    let currentPurchaseCount = 0;
+    let lottoList = [];
+    
+    while (currentPurchaseCount < this.purchaseCount) {
+      lottoList.push(Computer.buyLotto());
+      currentPurchaseCount += 1;
+    }
+    
+    return lottoList;
+  }
+
+  static buyLotto(){
+    return new Lotto(Random.pickUniqueNumbersInRange(1, 45, 6));
+  }
+
+  makeResultArray(lottoList, winNumbers, bonusNumber){
+    let resultArray = [];
+    lottoList.forEach((lotto) => {
+      resultArray.push(lotto.checkWin(winNumbers, bonusNumber));
+    });
```

Comment:
이런 로직에도 배열 메소드를 적용해서 간결하게 표현해볼 수 있을 것 같습니다 :)
```suggestion
const resultArray = lottoList.map((lotto) => checkWin(winNumbers, bonusNumber));
```

Reply:
resultArray와 같이 배열을 선언할때 const 키워드로 선언해주면 어떨까요? push 같은 메소드를 사용해서 요소를 추가하는 것은 const로 선언해도 가능하니 let을 사용하지 않아도 될 것 같아서요!